### PR TITLE
Add rc files for smooth running of Node stuff

### PR DESCRIPTION
**Pull request by shaundon**

### CODE REVIEW

**Review by scottandersonSkyscanner — APPROVED** [review]

File: .naverc [hunk 1]
```diff
@@ -0,0 +1 @@
+lts/carbon
```
Comment (georgegillams) [inline .naverc:1]: I find it better to put a specific version as the names are somewhat fluid

Comment (shaundon) [reply .naverc:1]: `lts/carbon` == `8.x.x`. I'm gonna leave it as this matches what we have in Backpack.

Comment (matthewdavidson) [reply .naverc:1]: The motivation behind relying on the aliases is that they are intended to be stable with only bug fixes and security patches. And FYI the current LTS is `lts/dubnium` ...

Comment (shaundon) [reply .naverc:1]: Ah yes, Dubnium, the element from which we produce dubstep music

Comment (georgegillams) [reply .naverc:1]: Ok cool 👍

**Pull request merged**